src/components/baseComponents/LanguageSelection.jsx

```diff
@@ -8,34 +8,31 @@ const LANGUAGES = [
   { value: 'de', label: 'Deutsch' },
 ];
 
-const LanguageSelection = () => {
+const LanguageSelection = ({ onChange }) => {
```

Comment:
This parameter is unused now that you're using a single useEffect hook to set local storage!